Jenkinsfile

```diff
@@ -33,6 +33,48 @@ pipeline {
       }
     }
 
+    stage('L2: BERT Squad v1.1') {
+      when {
+        anyOf{
+          branch 'candidate'
+          changeRequest()
+        }
+      }
+      failFast true
+        steps {
+          sh 'cd examples/nlp && CUDA_VISIBLE_DEVICES=0 python question_answering_squad.py --precision 16 --amp_level=O1 --train_file /home/TestData/nlp/squad_mini/v1.1/train-v1.1.json --eval_file /home/TestData/nlp/squad_mini/v1.1/dev-v1.1.json --batch_size 8 --gpus 1 --do_lower_case --pretrained_model_name bert-base-uncased --optimizer adamw --lr 5e-5 --max_steps 50 --scheduler WarmupAnnealing'
```

Comment:
Thanks for adding tests! 
Do we really need 50 and 10 steps? Can we do say 5 and/or just 2 to have reasonable confidence that nothing crashes?

Reply:
ok